src/js/components/FormField/FormField.js

```diff
@@ -49,6 +49,21 @@ const validateField = (required, validate, messages) => (value, data) => {
 };
 
 const FormFieldBox = styled(Box)`
+  ${props =>
+    props.label &&
+    props.label.hidden &&
+    // Visually hiding label, but leaving in DOM for screen readers.
+    // For background on approach see https://www.w3.org/WAI/tutorials/forms/labels/#note-on-hiding-elements
+    `label {
```

Comment:
Is this selector specific enough? For instance, if I have custom children of the FormField that includes label elements for other things, I don't think this should match those.

Instead, how about doing something like `const LabelText = styled(Text)` and use that?

Reply:
Implemented this suggestion.